src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -179,22 +179,34 @@ export class SearchResults extends Component {
 
     return (
       <>
-        <div className="find-forms-search-metadata vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row medium-screen:vads-u-justify-content--space-between">
+        <div
+          className="find-forms-search-metadata vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row medium-screen:vads-u-justify-content--space-between"
+          aria-atomic="false"
+          aria-live="assertive"
```

Comment:
@zacharymorel , question for you. Would it be possible to have an empty aria-live region div on the page _before_ the user searches for a form? If the `h2` search result is added into an already existing empty aria-live region, `aria-live="polite"` _should_ work. Otherwise, it may hiccup or act unpredictably as you described earlier.

> The ARIA live region must be present and must be empty before making an announcement. The best approach is to ensure the ARIA live region is already on the page when the page loads. You cannot insert an ARIA live region and make an announcement at the same time. The region needs to be on the page, and the accessibility API needs to be aware that it is there and that it is empty before any announcements will be sent to the screen reader. If you insert the ARIA live region after page load, you must use JavaScript to delay (typically for at least one second) before sending any text into it to make an announcement.

Apologies in advance if I read the code wrong and this behavior has already been tested 🙇

Reply:
Hey @joshkimux have you tested this locally? I think what we have works decent on both VO safari and chrome?

Reply:
@zacharymorel I don't have access just yet as I still have to set up SOCKS (doing this next week), so as of now I've only been able to review the code. Is there any chance you could pass this through staging for me to audit?

Reply:
Yes, I can merge this after @kelsonic || @ncksllvn approval.